src/DataContainer/DcaFilterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\DataContainer;
+
+/**
+ * DCA filter interface.
```

Comment:
What about this:

> Allows widgets that implement this interface to return a configuration array. In Contao
this is used for DataContainers that support the picker mode and thus might behave
differently depending on the settings of the widget. The widget implementing this interface must return an array of configuration settings. If the DataContainer supports these, they can work together.
See implementations of FileTree and DC_Folder for example.